src/nvim/tui/tui.c

```diff
@@ -55,11 +55,7 @@
 #define STARTS_WITH(str, prefix) (strlen(str) >= (sizeof(prefix) - 1) \
     && 0 == memcmp((str), (prefix), sizeof(prefix) - 1))
 #define TMUX_WRAP(is_tmux, seq) ((is_tmux) \
-                                 ? DCS_STR "tmux;\x1b" seq STERM_STR : seq)
-#define SCREEN_TMUX_WRAP(is_screen, is_tmux, seq) \
-    ((is_screen) \
-     ? DCS_STR seq STERM_STR : (is_tmux) \
-     ? DCS_STR "tmux;\x1b" seq STERM_STR : seq)
+    ? "\x1bPtmux;\x1b" seq "\x1b\\" : seq)
```

Comment:
```
  src/nvim/tui/tui.c:58:  Inner expression should be aligned as opening brace + 1 (+ 2 in case of {). Relevant opening is on line 57  [whitespace/alignment] [2]
```

Reply:
I think it's fixed now? Can't find the CI result.